packages/devtools_app/lib/src/screens/inspector_v2/widget_details.dart

```diff
@@ -35,44 +38,78 @@ class _WidgetDetailsState extends State<WidgetDetails> with AutoDisposeMixin {
 
   @override
   Widget build(BuildContext context) {
-    return ValueListenableBuilder<InspectorTreeNode?>(
-      valueListenable: controller.selectedNode,
-      builder: (context, _, __) {
-        final node = selectedNode;
-        if (node == null) {
-          return const Center(
-            child: Text(
-              'Select a widget to view its layout.',
-              textAlign: TextAlign.center,
-              overflow: TextOverflow.clip,
-            ),
-          );
-        }
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        final parentHeight = constraints.maxHeight;
+        final parentWidth = constraints.maxWidth;
+        final horizontalLayout = parentWidth >= (parentHeight * 1.25);
```

Comment:
What are these magic numbers here and below? Maybe they should be constants with comments explaining why they were chosen.

Reply:
Done!